### PR TITLE
PLAT-141820: Fix the issue that cannot find userSetupFile

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Fix the issue that cannot find userSetupFile

### Resolution
* Fix the typing error in userSetupFile variable in `./config/jest/jest.config.js`

### Links
PLAT-141820

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by hong6316** [comment]
I will add a comment in CHANGELOG.md after some PR will be merged.
It may harm to occur a conflict issue.

**Review by MikyungKim — COMMENTED** [review]
change log?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## unreleased
+
+### test
+
+* Fixed `./config/jest/jest.config.js` cannot reading userSetupFile that exists.
```
Comment (MikyungKim) [inline CHANGELOG.md:5]:
```suggestion
* Fixed cannot read a local `./src/setupTests.js` setup file.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## unreleased
+
+### test
+
+* Fixed cannot read a local `./src/setupTests.js` setup file.
```
Comment (MikyungKim) [inline CHANGELOG.md:5]:
```suggestion
* Fixed not reading a local `./src/setupTests.js` setup file.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 ### test
 
+* Fixed not reading a local `./src/setupTests.js` setup file.
+
```
Comment (MikyungKim) [inline CHANGELOG.md:6]: Please remove the empty line.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**